ext/bz2/bz2.stub.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @return resource|false
+ */
+function bzopen (string $filename, string $mode) {}
```

Comment:
Signature should be:
```suggestion
/**
 * @param string|resource $file
 * @return resource|false
 */
 function bzopen($file, string $mode) {}
```